[CI] LocalIndicesCleanerTests.testHandlesWatcherHistory fails reproducibly

This test has failed a few times no on `master`:

```
org.elasticsearch.xpack.monitoring.cleaner.local.LocalIndicesCleanerTests > testHandlesWatcherHistory FAILED
    java.lang.AssertionError: 
    Expected: <7>
         but: was <6>
        at __randomizedtesting.SeedInfo.seed([4A140D21CCAC26DF:F8559B4CC0706069]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.junit.Assert.assertThat(Assert.java:956)
        at org.junit.Assert.assertThat(Assert.java:923)
        at org.elasticsearch.xpack.monitoring.cleaner.local.LocalIndicesCleanerTests.lambda$assertIndicesCount$0(LocalIndicesCleanerTests.java:61)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:881)
        at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:854)
        at org.elasticsearch.xpack.monitoring.cleaner.local.LocalIndicesCleanerTests.assertIndicesCount(LocalIndicesCleanerTests.java:54)
        at org.elasticsearch.xpack.monitoring.cleaner.local.LocalIndicesCleanerTests.testHandlesWatcherHistory(LocalIndicesCleanerTests.java:79)
REPRODUCE WITH: ./gradlew ':x-pack:plugin:monitoring:test' --tests "org.elasticsearch.xpack.monitoring.cleaner.local.LocalIndicesCleanerTests.testHandlesWatcherHistory" -Dtests.seed=4A140D21CCAC26DF -Dtests.security.manager=true -Dtests.locale=es-CL -Dtests.timezone=America/Caracas -Dcompiler.java=13
```

The example above reproduced for me locally.

https://gradle-enterprise.elastic.co/s/qevr265r5v2bo/tests/iu7lux6ygquas-ar7w2ggk5jvdw

--- a/x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/cleaner/local/LocalIndicesCleanerTests.java
+++ b/x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/cleaner/local/LocalIndicesCleanerTests.java
@@ -57,12 +57,11 @@ public class LocalIndicesCleanerTests extends AbstractIndicesCleanerTestCase {
             //so when es core gets the request with the explicit index name, it throws an index not found exception as that index
             //doesn't exist anymore. If we ignore unavailable instead no error will be thrown.
             GetSettingsResponse getSettingsResponse = client().admin().indices().prepareGetSettings()
-                    .setIndicesOptions(IndicesOptions.fromOptions(true, true, true, true)).get();
+                    .setIndicesOptions(IndicesOptions.fromOptions(true, true, true, true, true)).get();
             assertThat(getSettingsResponse.getIndexToSettings().size(), equalTo(count));
         });
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/53025")
     public void testHandlesWatcherHistory() throws Exception {
         internalCluster().startNode();
 

